modules/labs-react/common/spec/useThemeRTL.spec.ts

```diff
@@ -1,18 +1,17 @@
-import {css} from '@emotion/core';
+import {css, CSSObject} from '@emotion/core';
 import {renderHook} from '@testing-library/react-hooks';
 import {useThemeRTL} from '../lib/theming';
 
 describe('styles', () => {
   describe(`useThemeRLT hook`, () => {
     const LTRTheme = {
-      canvas: {
-        direction: 'ltr',
-      },
+      direction: {direction: 'ltr'},
```

Comment:
I'm surprised this doesn't yell about not using `ContentDirection.LTR` here.

Reply:
I personally don't care either way. I'm just surprised.

Reply:
ah prob cause LTRTheme isn't a `partialTheme` type